app.py

```diff
@@ -0,0 +1,62 @@
+import time
+from openai import OpenAI
+import os
+
+# gets API Key from environment variable OPENAI_API_KEY
+client = OpenAI(
+    api_key= os.getenv("OPENAI_API_KEY"),
+)
+
+policyAssistant = client.beta.assistants.retrieve("asst_E2NTtF5sSbmxT10NvJk9WquB")
+skillAssistant = client.beta.assistants.retrieve("asst_6SndT0pLTqkvJTtCcV8Sfaxd")
+
+thread = client.beta.threads.create()
+
+def runAssistant(assistant_id,thread_id,user_instructions):
+    
+    run = client.beta.threads.runs.create(
+        thread_id=thread_id,
+        assistant_id=assistant_id,
+        instructions=user_instructions,
+    )
+
+    while True:
+        run = client.beta.threads.runs.retrieve(thread_id=thread.id, run_id=run.id)
+
+        if run.status == "completed":
+            print("This run has completed!")
+
+            break
+        else:
+            print("in progress...")
+            time.sleep(5)
+
+def handle_user_query():
+    while True:
+        user_query = input("Please enter your question (type 'exit' to quit): ")
+
+        if user_query.lower() == "exit":
+            break
+
+        if "skill" in user_query.lower():
```

Comment:
I don't think we can rely on this. What happens if I ask question like

```
Who are our .NET resources?
How many hours do I have to work at office?
```

Just checking for skill and policy is not enough. We needed more intelligence here. Why not keep an assistant to ask that? which we already discussed over call.

Reply:
Updated the classification logic 

![image](https://github.com/OsmosysSoftware/dynamic-assistant-poc/assets/121787291/09f23449-f294-4d69-8640-e40e6e469474)